### PR TITLE
Building EQ Example XML

### PR DESCRIPTION
**Pull request by laurenliz22**
Building EQ Example XML

#### Any background context you want to provide?
Adding Building EQ Example

#### What does this PR do?
Adding Building EQ Example

#### How should this be manually tested?
Run Building EQ Example

#### What are the relevant tickets?
N/A

### CODE REVIEW

**Review by JieXiong9119 — APPROVED** [review]
Looks good!

**Review by kflemin — APPROVED** [review]

**Review by JieXiong9119 — CHANGES_REQUESTED** [review]
One minor change required on <auc:UsefulLife>

File: examples/BuildingEQ-1.0.0.xml [hunk 1]
```diff
@@ -0,0 +1,2357 @@
...
+          </auc:TechnologyCategories>
+          <auc:MeasureScaleOfApplication>Entire building</auc:MeasureScaleOfApplication>
+          <auc:LongDescription>Remove Gas Ovens</auc:LongDescription>
+          <auc:MeasureSavingsAnalysis>
+            <auc:CalculationMethod>
+              <auc:Estimated/>
+            </auc:CalculationMethod>
+            <auc:OMCostAnnualSavings>260.0</auc:OMCostAnnualSavings>
+          </auc:MeasureSavingsAnalysis>
+          <auc:UsefulLife>50.0</auc:UsefulLife>
```
Comment (laurenliz22) [inline examples/BuildingEQ-1.0.0.xml:1558]:
```suggestion
          <auc:UsefulLife>50</auc:UsefulLife>
```

**Review by JieXiong9119 — APPROVED** [review]
Nice! And we will need to change this accordingly in the use case and the website repos

**Pull request merged**